frontend/ui/lib/services/places_services.dart

```diff
@@ -0,0 +1,2 @@
+export 'package:tripmeout/services/places_services_stub.dart'
+    if (dart.library.html) 'package:tripmeout/services/places_services_html.dart';
```

Comment:
Consider using a platform-independent library that allows this to be tested locally. As of now, tests will _always_ call the stub code.